src/uthash.h

```diff
@@ -714,37 +699,28 @@ typedef unsigned char uint8_t;
         hashv += (unsigned)(keylen);                                                                         \
         switch (_hj_k) {                                                                                     \
             case 11:                                                                                         \
-                hashv += ((unsigned)_hj_key[10] << 24);                                                      \
-                H5_ATTR_FALLTHROUGH                                                                          \
+                hashv += ((unsigned)_hj_key[10] << 24); /* FALLTHROUGH */                                    \
```

Comment:
Did you want to re-insert the H5_ATTR_FALLTHROUGH?

Reply:
No, he uses /* FALLTHROUGH */ comments, which most compilers accept as intent. This is mentioned in the changelogs.

Reply:
OK, great